src/Sylius/Bundle/CoreBundle/Fixture/Factory/ProductExampleFactory.php

```diff
@@ -391,4 +392,15 @@ private function getRandomValueForProductAttribute(ProductAttributeInterface $pr
                 throw new \BadMethodCallException();
         }
     }
+
+    private function generateProductVariantName(ProductVariantInterface $variant): string
+    {
+        return trim(array_reduce(
+            $variant->getOptionValues()->toArray(),
+            static function(string $variantName, ProductOptionValueInterface $variantOption) {
```

Comment:
```suggestion
            static function(?string $variantName, ProductOptionValueInterface $variantOption) {
```
And removing the third argument? Not sure if it won't be more obvious what is happening here :)